metricbeat/beater/event.go

```diff
@@ -15,31 +15,31 @@ const (
 // eventBuilder is used for building MetricSet events. MetricSets generate a
 // data in the form of a common.MapStr. This builder transforms that data into
 // a complete event and applies any Module-level filtering.
-type eventBuilder struct {
-	moduleName    string
-	metricSetName string
-	host          string
-	startTime     time.Time
-	fetchDuration time.Duration
-	event         common.MapStr
+type EventBuilder struct {
+	ModuleName    string
+	MetricSetName string
+	Host          string
+	StartTime     time.Time
+	FetchDuration time.Duration
+	Event         common.MapStr
 	fetchErr      error
 	filters       *processors.Processors
 	metadata      common.EventMetadata
 }
 
 // build builds an event from MetricSet data and applies the Module-level
```

Comment:
Consider diff'ing a before and after `golint ./... | grep -v vendor` output to look for new warnings.